package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "hyper-content-for",
+  "version": "0.1.0",
+  "description": "Allows you to store HTML for use (and re-use) later.",
+  "main": "hyper-content-for.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/hyperoslo/hyper-content-for-angular.git"
+  },
```

Comment:
You can use npm's shortcut for GitHub repositories (https://github.com/timkurvers/blizzardry/blob/master/package.json#L13):

```json
"repository": "timkurvers/blizzardry"
```

Which will automatically set `repository`, `bugs` and `homepage`.